go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -488,65 +543,91 @@ func newSharedGatewayOpenFlowManager(patchPort, macAddress, gwBridge, gwIntf str
 
 	// add health check function to check default OpenFlow flows are on the shared gateway bridge
 	ofm := &openflowManager{
-		gwBridge:    gwBridge,
-		physIntf:    gwIntf,
-		patchIntf:   patchPort,
-		ofportPhys:  ofportPhys,
-		ofportPatch: ofportPatch,
-		flowCache:   make(map[string][]string),
-		flowMutex:   sync.Mutex{},
-		flowChan:    make(chan struct{}, 1),
+		gwBridge:        gwBridge.bridgeName,
+		physIntf:        gwBridge.uplinkName,
+		patchIntf:       patchPort,
+		exGWGwBridge:    exGWBridge.bridgeName,
```

Comment:
this is an awkward name, can it just be exGWBridge? Also you set on line 566 exGWBridge, which I'm not seeing here...